components/Greeting/Greeting.tsx

```diff
@@ -0,0 +1,7 @@
+import { auth } from '@/auth';
+
+export default async function Greeting() {
+  const session = await auth();
+  const firstName = session?.user?.name?.split(' ')[0];
+  return <h2 className='text-4xl'>Hello {firstName}!</h2>;
+}
```

Comment:
the className inline styling is proper tailwindCSS usage, it is a better solution in case of server components in comparison to CSS-in-JS like styled-components which run at run-time istead of build-time like tailwind does.